lib/serverless-tools/cli.rb

```diff
@@ -13,9 +13,10 @@ def comment
     end
 
     desc "deploy", "publishes and deploys the specified lambda functions"
-    method_option :filename, :type => :string, :aliases => "-f"
+    method_option :filename, :type => :string, :aliases => "-f", :default => "functions.yml"
+    method_option :force, :type => :boolean, :aliases => "-x", :default => false
```

Comment:
Why do we need to specify defaults here as well as in Deployer.deploy? Does it give some nice command line output or something?

Reply:
They're probably not needed in the Deployer.deploy method - but I think it's nice to have sensible defaults for testing them independently.